### PR TITLE
feat/set up Storybook

### PR DESCRIPTION
**Pull request by filwas**
This is a second attempt at adding a ui-system package module with Storybook. This time, we're using Next.js immediately, so we'll have an easier job migrating the components.

### CODE REVIEW

**Review by Sumick — APPROVED** [review]
Very good job - everything is placed very well. You did great guys 🫡 Only one thing to fix before we can merge it - let's remove images from `ui-system`, so we do not import them to the platform.

**Review by Wiecek-K — APPROVED** [review]

File: packages/ui-system/src/components/Logo/Logo.tsx [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
+import styles from './Logo.module.scss'
 
-const Logo = () => {
+export const Logo = () => {
   return (
     <Link href={AppRoutes.home} className={styles.logo}>
       <img src={logo.src} alt="Logo" />
       <div className={styles.title}>Good Dev Hunting</div>
     </Link>
   )
```
Comment (mmcode9407) [inline packages/ui-system/src/components/Logo/Logo.tsx:12]: We should pass some props to the component, eg. href, src, alt, title.

File: packages/ui-system/src/app/page.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
+import Image from "next/image";
+import styles from "./page.module.css";
+import { Button } from "@/components/Button/Button";
+
+export default function Home() {
+  return (
+    <main className={styles.main}>
+      <Button variant={"primary"}> chuj</Button>
```
Comment (mmcode9407) [inline packages/ui-system/src/app/page.tsx:8]: Let's remove 'chuj' 😁

File: apps/platform-app/src/app/[locale]/(profile)/(components)/Header/Header.tsx [hunk 3]
```diff
@@ -134,7 +134,7 @@ async function Header({ buttonsVariant = 'main' }: HeaderProps) {
               <img src={logo.src} alt="Logo" />
               <h1 className={styles.title}>Good Dev Hunting</h1>
             </Link>
```
Comment (mmcode9407) [inline apps/platform-app/src/app/[locale]/(profile)/(components)/Header/Header.tsx:136]: Let's use `Logo` from ui-system

**Review by mmcode9407 — APPROVED** [review]
I left few comments before merging.

And remove logo.png from ui-system when you add props to Logo component :)

**Pull request merged**